packages/apps-config/src/api/bundle/spec/index.ts

```diff
@@ -6,5 +6,6 @@ import laminar from './laminar';
 
 export default {
   acala,
-  laminar
+  laminar,
+  mandala: acala
```

Comment:
we changed the spec name from `acala` to `mandala`, keep both for compatibility with old testnet